src/solvers/dgmulti/sbp.jl

```diff
@@ -257,4 +257,82 @@ function StartUpDG.inverse_trace_constant(rd::RefElemData{NDIMS, ElementType, Ap
   #     Chan, Jesse, et al (2016), https://doi.org/10.1016/j.jcp.2016.04.003
   # for more details (specifically, Appendix A.1, Theorem A.4).
   return NDIMS * max_eigenvalue
-end
\ No newline at end of file
+end
+
+# type alias for specializing on a periodic SBP operator
+const DGMultiPeriodicFDSBP{ApproxType, ElemType} =
+  DGMulti{NDIMS, ElemType, ApproxType, SurfaceIntegral, VolumeIntegral} where {NDIMS, ElemType, ApproxType<:SummationByPartsOperators.PeriodicDerivativeOperator, SurfaceIntegral, VolumeIntegral}
+
+"""
+  CartesianMesh(dg::DGMultiPeriodicFDSBP{NDIMS})
+
+Constructs a single-element `mesh::AbstractMeshData` for a single periodic element given
+a DGMulti with approximation_type <: SummationByPartsOperators.AbstractDerivativeOperator.
+"""
+function CartesianMesh(dg::DGMultiPeriodicFDSBP{NDIMS}) where {NDIMS}
+
+  rd = dg.basis
+
+  e = ones(size(rd.r))
+  z = zero.(e)
```

Comment:
```suggestion
  z = zero(e)
```
Or does this simpler version without broadcasting do something different than desired?